<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="showSub">
      <property name="text">
       <string>Show SubWindow</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addNode">
      <property name="text">
       <string>Add Node</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_1">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/wenha/Desktop/add.png</normaloff>C:/Users/wenha/Desktop/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">modeGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/wenha/Desktop/62308.png</normaloff>C:/Users/wenha/Desktop/62308.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">modeGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_3">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/wenha/Desktop/colorful_31488.png</normaloff>
        <selectedon>pictures/add.png</selectedon>C:/Users/wenha/Desktop/colorful_31488.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">modeGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_4">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/wenha/Desktop/62308_inv.png</normaloff>
        <selectedon>pictures/delete.png</selectedon>C:/Users/wenha/Desktop/62308_inv.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonIconOnly</enum>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">modeGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="modeGroup"/>
 </buttongroups>
</ui>
